### ISSUE TITLE
loss is descending,but the val loss=nan???

### ISSUE THREAD

**Issue by hexin6688** [qqwweee/keras-yolo3#659]
loss is descending,but the val loss=nan,anybody have face this problem?

**Comment by kartikwar** [comment]
Yes I faced it too..when I tried with tiny yolo model and custom anchors

**Comment by BBarbosa** [comment]
I did face this issue for _val_loss_ when disabled the random flag on the validation data generator

**Comment by hexin6688** [comment]
when trainning on the first stage,the val
 loss=nan,but when training on the second stage(unfreezing) the val loss starts to descending.

**Comment by Zhang-KL** [comment]
Have you solved your problem? What is the reason?
